fastlane/lib/fastlane/actions/docs/upload_to_play_store.md

```diff
@@ -83,6 +83,18 @@ Expansion files (obbs) found under the same directory as your APK will also be u
 - they are identified as type 'main' or 'patch' (by containing 'main' or 'patch' in their file name)
 - you have at most one of each type
 
+If you want only update expansion files from previous version on Google Play use
```

Comment:
As this now increases the information on obbs, can we add a headline in line 80 please?
```
### Expansion files (`.obb`)
```